posthog/async_migrations/test/__snapshots__/test_0004_replicated_schema.ambr

```diff
@@ -101,126 +101,24 @@
   )
 ---
 # name: Test0004ReplicatedSchema.test_migration.6
-  <class 'tuple'> (
-    'kafka_groups',
-    "Kafka('kafka', 'clickhouse_groups_test', 'group1', 'JSONEachRow')",
-  )
----
-# name: Test0004ReplicatedSchema.test_migration.7
   <class 'tuple'> (
     'kafka_events_json',
     "Kafka('kafka', 'clickhouse_events_json_test', 'group1', 'JSONEachRow')",
   )
 ---
-# name: Test0004ReplicatedSchema.test_migration.8
-  <class 'tuple'> (
```

Comment:
What's going on here?

Reply:
this branches off from `nuke-protobuf-pt1` because of the kafka json table (notice the base). I ran the snapshot update here instead of there hence this noise. Fixed now.